src/osd/OSD.cc

```diff
@@ -896,6 +896,22 @@ void OSDService::share_map_peer(int peer, Connection *con, OSDMapRef map)
   }
 }
 
+bool OSDService::check_scrubs_pending()
```

Comment:
This looks like a duplicate of OSDService::inc_scrubs_pending, does it make sense to use an argument to distinguish whether it is a test, or do actual increasing of the pending?